tests/test_gates.py

```diff
@@ -663,3 +663,18 @@ def circuit():
             circ = qml.QNode(circuit, dev)
             circ_def = qml.QNode(circuit, dev_def)
             assert np.allclose(circ(), circ_def(), tol)
+
+@pytest.mark.parametrize("num_qubits", [2, 3, 4])
+def test_qft_gate(num_qubits):
+    dev = qml.device("lightning.qubit", wires=num_qubits)
```

Comment:
Just a note before we start the actual review. Please use `default.qubit` in this pytest to check the validity of the new implementation.

Reply:
Does the latest change test your implementation against `default.qubit`? This is what I meant.
You may want to take a look at the other tests in this file.

Reply:
yes